SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -257,19 +257,20 @@ void USpatialActorChannel::DeleteEntityIfAuthoritative()
 
 	if (bHasAuthority)
 	{
+		check(Actor != nullptr);
 		// Workaround to delay the delete entity request if tearing off.
 		// Task to improve this: https://improbableio.atlassian.net/browse/UNR-841
-		if (Actor != nullptr && Actor->GetTearOff())
+		if (Actor->GetTearOff())
 		{
-			NetDriver->DelayedSendDeleteEntityRequest(EntityId, 1.0f);
+			NetDriver->DelayedSendDeleteEntityRequest(EntityId, 1.0f, Actor->IsNetStartupActor());
```

Comment:
This `1.0f` seems abritrary and should at least be a constant with a comment on why it's this size

Reply:
+1 although I'm guessing this is 1.0 because the old code was. And I suspect it did it as a hack. Just delaying deletion seems like something that will bite us later but fixing that seems out of scope.

Reply:
yeah this is old code and detailed in the UNR-841 ticket mentioned above